### ISSUE TITLE
mupx mongoimport & mongoexport

### ISSUE THREAD

**Issue by markshust** [arunoda/meteor-up-legacy#699]
I don't think it's possible to do a mongoimport or mongoexport. Do you think it would be a good idea to create a command such as `mupx mongoimport --file table.json` that runs as a one-off container to import a file? Same for export?

One would just need to decide where to place the import/export files and where the container would look for these.

**Comment by comigor** [comment]
I guess one could execute a command on mongodb's docker instance `docker exec -it mongodb mongoexport ...` or expose the mongodb's port locally and execute the same command on the server.

**Comment by occasl** [comment]
I personally would find such functionality useful but I suppose ssh'ing into the EC2 instance is fine too to execute a docker command but then I would have to deal with the logistics of getting the import files up there. Ideally, it would be nice to just refer to the files locally from my machine and `mupx` could handle scp'ing them to the server.

**Comment by markshust** [comment]
@occasl agree, that would be a great workflow. so running for example `mupx mongoimport --file mydb` would scp `mydb` to a one-off container and execute the import. it would be nice if exports were handled the same way.

**Comment by comigor** [comment]
Yeah, you guys are right. It would be handy to have this functionality!
